demo/app-root.ts

```diff
@@ -1,21 +1,134 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+/* eslint-disable no-restricted-globals */
+import {
+  html,
+  css,
+  LitElement,
+  customElement,
+  property,
+  query,
+} from 'lit-element';
+import '../src/item-inspector/item-inspector';
+import {
+  MetadataResponse,
+  SearchService,
+} from '@internetarchive/search-service';
+import '../src/item-navigator';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @property({ type: Object }) itemMD: MetadataResponse | undefined = undefined;
```

Comment:
quick question: why do we have `undefined = undefined;`  here, can't we just assign an empty `{}` instead or something?

Reply:
You can shorthand this to

```suggestion
  @property({ type: Object }) itemMD?: MetadataResponse;
```

It will default to `undefined`

Reply:
the component itself fetches the metadata so i want to keep the true state.